tools/benchmark/README.md

```diff
@@ -47,7 +47,7 @@ Notice that the benchmark_app usually produces optimal performance for any devic
 $benchmark_app -m <model> -i <input> -d CPU
 ```
 
-But it is still may be non-optimal for some cases, especially for very small networks. More details can read in [Introduction to Performance Topics](./docs/IE_DG/Intro_to_Performance.md).
+But it is still may be non-optimal for some cases, especially for very small networks. More details can read in [Introduction to Performance Topics](https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_DG/Intro_to_Performance.md).
```

Comment:
Please change the link to the following: 
`[Introduction to Performance Topics](@ref openvino_docs_IE_DG_Intro_to_Performance)`

Reply:
@x1aoo ping